ibexa/commerce/4.0.x-dev/manifest.json

```
@@ -127,36 +127,38 @@
-    "#5": "When using redis or memcached adapters, you can configure it using DSN:",
-    "#6": "- https://symfony.com/doc/4.4/components/cache/adapters/redis_adapter.html#configure-the-connection",
-    "#7": "- https://symfony.com/doc/4.4/components/cache/adapters/memcached_adapter.html#configure-the-connection",
+    "#7": "When using redis or memcached adapters, you can configure it using DSN:",
+    "#8": "- https://symfony.com/doc/4.4/components/cache/adapters/redis_adapter.html#configure-the-connection",
+    "#9": "- https://symfony.com/doc/4.4/components/cache/adapters/memcached_adapter.html#configure-the-connection",
     "CACHE_DSN": "localhost",
     "CACHE_NAMESPACE": "ezp",
     "DATABASE_CHARSET": "utf8mb4",
     "DATABASE_COLLATION": "utf8mb4_unicode_520_ci",
-    "#8": "Needed by Doctrine Bundle / ORM to avoid it opening connection during situations where there is no service yet.",
-    "#9": "See: https://symfony.com/doc/current/reference/configuration/doctrine.html#doctrine-dbal-configuration",
+    "#10": "Needed by Doctrine Bundle / ORM to avoid it opening connection during situations where there is no service yet.",
+    "#11": "See: https://symfony.com/doc/current/reference/configuration/doctrine.html#doctrine-dbal-configuration",
     "DATABASE_VERSION": "mariadb-10.3.0",
     "SESSION_HANDLER_ID": "session.handler.native_file",
     "SESSION_SAVE_PATH": "%kernel.project_dir%/var/sessions/%kernel.environment%",
     "SOLR_CORE": "collection1",
-    "#10": "Enable recommendations by setting valid id, key and uri",
+    "#12": "Enable recommendations by setting valid id, key and uri",
     "RECOMMENDATION_CUSTOMER_ID": 0,
     "RECOMMENDATION_LICENSE_KEY": "",
     "RECOMMENDATION_HOST_URI": "",
     "ELASTICSEARCH_DSN": "http://localhost:9200",
-    "#11": "Enable fastly by setting valid service_id and key",
-    "#12": "In order for this to work you also need to have EzSystemsPlatformFastlyCacheBundle installed",
-    "#13": "FASTLY_SERVICE_ID=\"\"",
-    "#14": "FASTLY_KEY=\"\"",
+    "#13": "Enable fastly by setting valid service_id and key",
+    "#14": "In order for this to work you also need to have EzSystemsPlatformFastlyCacheBundle installed",
+    "#15": "FASTLY_SERVICE_ID=\"\"",
+    "#16": "FASTLY_KEY=\"\"",
     "SISO_SEARCH_SOLR_HOST": "localhost",
     "SISO_SEARCH_SOLR_PORT": 8983,
     "SISO_SEARCH_SOLR_CORE": "collection1",
-    "#15": "Do not add 'solr' to the path. It is automatically added by SolariumBundle",
+    "#17": "Do not add 'solr' to the path. It is automatically added by SolariumBundle",
     "SISO_SEARCH_SOLR_PATH": "",
-    "#16": "SolariumBundle Solr DSN configuration",
-    "#17": "Note that /solr suffix will be appended autmatically to eZ Platform Solr Search Engine Bundle configuration",
+    "#18": "SolariumBundle Solr DSN configuration",
+    "#19": "Note that /solr suffix will be appended autmatically to eZ Platform Solr Search Engine Bundle configuration",
```

```suggestion
    "#19": "Note that the /solr suffix will be appended automatically to Ibexa Solr Search Engine Bundle configuration",
```
I guess, given it's 4.0